lang/nl.po

```diff
@@ -0,0 +1,39 @@
+msgid ""
+msgstr ""
+"Language: nl\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+msgid "msg_help_usage"
+msgstr "Gebruik: vrms-rpm [opties]"
+msgid "msg_help_option_ascii"    
+msgstr "    Toont ASCII-kunst wanneer er alleen vrije software gevonden is,\n"
+       "    of wanneer meer dan 10%% van de software niet vrij is."
+msgid "msg_help_option_explain"
+msgstr "    Toon licenties bij het classificeren van pakketten,\n"
+       "    om de vrij / niet-vrij classificatie te verduidelijken."
+msgid "msg_help_option_help"
+msgstr "    Toon deze help en stop."
+msgid "msg_help_option_list"
+msgstr "    Toon niet alleen het aantal vrije en niet-vrije pakketten,\n"
+       "    maar toon hun namen. De standaardwaarde is 'nonfree'."
+msgid "msg_help_option_version"
+msgstr "    Toon versie-informatie en stop."
+msgid "msg_list_noargument"
+msgstr "%s: --list optie vereist een argument."
+msgid "msg_list_badargument"
+msgstr "%s: het argument voor de optie --list moet 'none', 'free', 'nonfree' of 'all' zijn."
+msgid "msg_unknown_option"
+msgstr "%s: onbekende optie '%s'"
+msgid "msg_total_free"
+msgstr "%d vrije paketten"
+msgid "msg_total_nonfree"
+msgstr "%d niet-vrije paketten"
+msgid "msg_rms_happy_ascii"
+msgstr "Enkel vrije paketten - rms zou trots zijn!                           rms is blij."
```

Comment:
This text should be exactly 79 characters long so it fits nicely on a standard terminal width.